.github/workflows/main.yml

```diff
@@ -11,30 +11,44 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-11, macos-12]
+        os: [macos-11, macos-12, ubuntu-latest]
```

Comment:
COPY --from=builder /opt/local /opt/local
COPY --from=builder /MacPorts-2.8.1 /MacPorts-2.8.1

ARG CONF_PATH=/opt/local/etc/macports/macports.conf
# Write some required defaults.
# Unless, macports-base is modified so that it is able to detect these, 
# we need to hardcode them

# 2 because this image is mostly intended to run on VMs
RUN echo "buildmakejobs 2" >> $CONF_PATH

RUN echo "cxx_stdlib libstdc++" >> $CONF_PATH
RUN echo "build_arch x86_64" >> $CONF_PATH

# add symlink for gnumake
RUN ln -s /usr/bin/make /usr/bin/gnumake

ENV PATH="/opt/local/bin:$PATH"

Reply:
> 
> COPY --from=builder /opt/local /opt/local
> COPY --from=builder /MacPorts-2.8.1 /MacPorts-2.8.1
> 
> ARG CONF_PATH=/opt/local/etc/macports/macports.conf
> # Write some required defaults.
> # Unless, macports-base is modified so that it is able to detect these, 
> # we need to hardcode them
> 
> # 2 because this image is mostly intended to run on VMs
> RUN echo "buildmakejobs 2" >> $CONF_PATH
> 
> RUN echo "cxx_stdlib libstdc++" >> $CONF_PATH
> RUN echo "build_arch x86_64" >> $CONF_PATH
> 
> # add symlink for gnumake
> RUN ln -s /usr/bin/make /usr/bin/gnumake
> 
> ENV PATH="/opt/local/bin:$PATH"

[192.168.1.1](diego-duy1@hotmail.com)